lib/fizzy/execute.hpp

```diff
@@ -20,23 +20,23 @@ struct execution_result
 
 struct Instance;
 
-using ImportedFunction = std::function<execution_result(Instance&, std::vector<uint64_t>)>;
+using ExternalFunction = std::function<execution_result(Instance&, std::vector<uint64_t>)>;
```

Comment:
maybe `external_function`?
There are `execution_result` and `Instance` in this file, so no consistency.